manifests/init.pp

```diff
@@ -23,9 +23,12 @@
         String                  $service,
     ) {
 
-    package { $packages:
-        ensure => installed,
-        notify => Service[$service],
+    # cron is part of the base OS, not a package
```

Comment:
Would it be possible to eliminate this condition by setting `cron::packages` to null in Hiera?

Reply:
That was my first thought as well however puppet will throw a "resource title resulted in undef" error if the value is set to null.  This also happens on other types of resources such as services.

Reply:
I understand wanting to avoid this conditional though, it could get messy adding in support for other operating systems.